.github/workflows/lint.yml

```diff
@@ -4,8 +4,17 @@ on: [push, pull_request, workflow_dispatch]
 
 jobs:
   pre-commit:
+    name: Run pre-commit
     runs-on: ubuntu-latest
+
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
-    - uses: pre-commit/action@v2.0.0
+      - name: Check out repo
```

Comment:
Maybe I missed it somewhere (please do point it out if so), but I can't seem to find where you've explained why it is actively harmful to do so, when it adds descriptiveness and clarity (as well as looks more polished) in the GitHub UI. The only thing I noticed you mention was that simply that you personally don't add it in your own work, which I don't see how has any bearing on blocking me from adding it here in mine, particularly when there are concrete (clarity, descriptiveness and clean UI) reasons to do otherwise and on a workflow I'm responsible for maintaining.

I don't personally care for the use of emojis in step names (and more practically, they don't show up at all in my editor), but it would be equally inappropriate for me to propose or request removing them from the workflow you added due my personal preference, and likewise I would be totally fine with including one of your choice in a new name I added, to be consistent with the others.